consumables/scss/components/unified-header/_left-nav-trigger.scss

```diff
@@ -11,6 +11,20 @@
   cursor: pointer;
   z-index: 9999;
 
+  &:focus {
+    outline: -webkit-focus-ring-color auto 5px;
```

Comment:
For consistency, can you use the focus-outline mixin? 

```scss
@import '../../global/helpers';

.bx--left-nav__trigger {
  ...

  &:focus {
    // outline: -webkit-focus-ring-color auto 5px;
    @include focus-outline;
  }
  ...
}
```

This just helps us to know where we're applying focus styles.